modules/apps/headless/headless-discovery/headless-discovery-web/src/js/APIGUI.js

```diff
@@ -298,6 +298,7 @@ const APIGUI = () => {
 							'delete',
 							'patch',
 						]}
+						tryItOutEnabled={true}
```

Comment:
I understand that with the version change the `tryItOutEnabled` is false by default?

Reply:
no, it didn't have that feature. We had something done manually by looking at the DOM and enabling it but we removed because it was very buggy.